### ISSUE TITLE
Move centroid_dashboard into mica

### ISSUE THREAD

**Issue by jeanconn** [sot/mica#203]

**Comment by taldcroft** [comment]
Reference for setup.py:
https://github.com/sot/eng_archive/blob/878be1b40a5022100a0c98d4d37398c01e0c7869/setup.py

```
console_scripts = ['mica_update_centroid_dashboard_archive = mica.centroid_dashboard:update_archive'
 'mica_make_centroid_dashboard_report = mica.centroid_dashboard:make_report']
```

**Comment by taldcroft** [comment]
Closed by #236
